composer/algorithms/blurpool/README.md

```diff
@@ -72,6 +72,11 @@ def train_model(model, train_dataloader):
 The Composer implementation of BlurPool uses model surgery to replace instances of pooling and downsampling operations with the BlurPool equivalents.
 For max pooling, it replaces `torch.nn.MaxPool2d` instances with instances of a custom `nn.Module` subclass that decouples the computation of the max within a given spatial window from the pooling and adds a spatial low-pass filter in between. This change roughly doubles the data movement required for the op, although it shouldn’t add significant overhead unless there are many maxpools in the network. For convolutions, it replaces strided `torch.nn.Conv2d` instances (i.e., those where the stride is larger than 1) with a custom module class that 1) applies a low-pass filter to the input, and then 2) applies a copy of the original convolution operation.
 
+🚧 Implementation Note
+>
+> Blurpool refuses to blur the input layer of a network (since at that point you're just reducing the input), as measured by whether the layer has [too few channels](https://github.com/mosaicml/composer/blob/4945b85dcaf65ce7f79ba9fdfd37b25a4b83a070/composer/algorithms/blurpool/blurpool.py#L169).
```

Comment:
We probably need more precise language here, cc: @dblalock

Reply:
"Blurpool does not replace strided convolutions with fewer than 16 input channels. This is a heuristic used to avoid blurpooling the network's input. Doing so is undesirable since it amounts to downsampling the input by more than the amount specified in the preprocessing pipeline."

Although in retrospect I wouldn't mind just removing this logic entirely. Seems more confusing than the extra blurring.

Reply:
Okay, I added the new language.